### PR TITLE
Add support for llama 3.1

### PR DESCRIPTION
**Pull request by rickzx**

### CODE REVIEW

File: llama3.1-70b-instruct-awq/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+accelerate==0.29.3
+autoawq==0.2.5
+bentoml>=1.2.17
+bitsandbytes==0.43.1
+fastapi==0.111.1
+openai==1.35.14
+vllm==0.5.3.post1; sys_platform == "linux"
```
Comment (larme) [inline llama3.1-70b-instruct-awq/requirements.txt:7]: let's stick with 0.5.0.post1 for now, or openai endpoints will break

**Review by larme — CHANGES_REQUESTED** [review]
Just update reqs.txt and other LGTM

File: llama3.1-70b-instruct-awq/requirements.txt (hunk 1 above)
Comment (rickzx) [reply llama3.1-70b-instruct-awq/requirements.txt:7]: It has to be updated to `0.5.3.post1` due to backward incompatible API change in llama model.
See: https://github.com/vllm-project/vllm/pull/6690
I fixed the openai endpoint. Does it not work?

**Review by larme — APPROVED** [review]
LGTM

**Pull request merged**